### DISCUSSION TITLE
ExponentialBucketHistogram

### DISCUSSION THREAD

**Discussion by StephenLiptrottTote** [open-telemetry/opentelemetry-dotnet#3757] [Q&A]
Hi,

What is the status of ExponentialBucketHistogram , was looking to know if its in a working state to use for metrics?
Currently its marked as an internal class is that correct?

**Comment by cijothomas** [comment]
Its internal now. @alanwest is looking to incorporate it to the main SDK, trying to hide it from public api until after 1.4 release (Nov 2022)

**Reply by StephenLiptrottTote** [reply]
Awesome thanks for the update!
